### PR TITLE
MLPAB-1425 Ask for confirmation certificate provider is not related

### PR DESCRIPTION
**Pull request by hawx**

### CODE REVIEW

File: internal/page/donor/confirm_your_certificate_provider_is_not_related.go [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+				if err := donorStore.Put(r.Context(), lpa); err != nil {
+					return err
+				}
+
+				return appData.Redirect(w, r, lpa, page.Paths.CheckYourLpa.Format(lpa.ID))
+			}
+		}
+
+		return tmpl(w, data)
```
Comment (acsauk) [inline internal/page/donor/confirm_your_certificate_provider_is_not_related.go:43]: Could we get some unit tests to cover these?

**Review by acsauk — COMMENTED** [review]
Good to go once the handler gets some coverage (unless there's a reason for leaving them out?)

File: internal/page/donor/confirm_your_certificate_provider_is_not_related.go (hunk 1 above)
Comment (hawx) [reply internal/page/donor/confirm_your_certificate_provider_is_not_related.go:43]: yep, I just totally missed that 😄

**Review by acsauk — APPROVED** [review]
:shipit:

**Pull request merged**